qiskit/circuit/library/n_local/n_local.py

```diff
@@ -207,7 +203,7 @@ def flatten(self, flatten: bool) -> None:
         self._invalidate()
         self._flatten = flatten
 
-    def _convert_to_block(self, layer: Any) -> QuantumCircuit:
+    def _convert_to_block(self, layer: typing.Any) -> QuantumCircuit:
```

Comment:
`Any` means "don't type check this", whereas `object` is the base of all Python objects.  The fact that this function takes either probably indicates that there's some usage of these objects that ought to be tightened up a bit, but it's not a big deal (and out of scope of this PR anyway).

Reply:
Yes, this can probably be changed to `Operation`